pandas/compat/numpy/function.py

```diff
@@ -134,7 +137,7 @@ def validate_argsort_with_ascending(ascending, args, kwargs):
     return ascending
 
 
-CLIP_DEFAULTS = dict(out=None)
+CLIP_DEFAULTS = dict(out=None)  # type Dict[str, None]
```

Comment:
Seems strange to have a value of None-type - is this not populated elsewhere?

Reply:
No value of `CLIP_DEFAULTS` is never changed. There are two other similar variables at around Line 210-220.

Reply:
@gfyoung any insight here? Are these variables actually unused or is this intentional?

Reply:
`CLIP_DEFAULTS` --> `validate_clip` --> `validate_clip_with_axis` --> `clip` (in `core/generic.py`)